spec/lib/dispatch-rider/queue_services/base_spec.rb

```diff
@@ -38,13 +38,15 @@
 
     context "when the block passed to process the popped message returns true" do
       it "should return the first message in the queue" do
-        simple_queue.pop {|msg| true}.should eq(DispatchRider::Message.new(:subject => 'foo', :body => 'bar'))
+        response = simple_queue.pop { |msg| true }
+        expect(response).to eq(DispatchRider::Message.new(:subject => 'foo', :body => 'bar'))
       end
     end
 
     context "when the block passed to process the popped message returns false" do
       it "should return the first message in the queue" do
-        simple_queue.pop {|msg| false}.should eq(DispatchRider::Message.new(:subject => 'foo', :body => 'bar'))
+        result = simple_queue.pop { |msg| false }
```

Comment:
Unused block argument - `msg`. You can omit the argument if you don't care about it.